DigitalLearningSolutions.Data.Migrations/202201111021_ChangeCompetencyAssessmentQuestionRoleRequirementsPrimaryKey.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace DigitalLearningSolutions.Data.Migrations
+{
+    using FluentMigrator;
+
+    [Migration(202201111021)]
+    public class ChangeCompetencyAssessmentQuestionRoleRequirementsPrimaryKey: Migration

```

Comment:
This was the original migration made when we wanted to replace the ID column with a compound PK on the two necessary columns. As it was pushed into an MR, and thus run on Jenkins, prior to the decision to just use a unique constraint instead, this migration has been left, and the other migration tackles it.

Reply:
Ah, missed this comment!